src/app/event/new-scheduled-event/new-scheduled-event.component.ts

```diff
@@ -199,7 +199,7 @@ export class NewScheduledEventComponent implements OnInit {
       if (this.simpleUserCounts[ea.environment]) {
         initVal = this.simpleUserCounts[ea.environment] || 0; // so we don't blow away old input values when rebuilding this form
       }
-      var newControl = new FormControl(initVal, [Validators.pattern(/-?\d+/), Validators.max(this.maxUserCounts[ea.environment])]);
+      var newControl = new FormControl(initVal, [Validators.pattern(/-?\d+/), Validators.max(this.maxUserCounts[ea.environment]), Validators.min(1)]);
```

Comment:
Does Validators.min(1) allow that one of the environments / vms is 0. 
If we have multiple environments or availabe vms in an environment this might brake and required the user to set each environment or vm to at least 1 machine.